README.md

```diff
@@ -50,16 +50,16 @@ All post requests are sent with the header "X-Disable-Provenance" set to "true".
 First, we need to create a folder.
```

Comment:
Maybe it could be nice to show the possibility to run a different script locally with a container:
`docker run admin-tools python delete_indexed_equipments.py -n` in addition to `docker run admin-tools` ?

Reply:
Nice will add it to the README